File: beginner/javascript/longest-word.js
/**
 * Given a sentence, return the longest word in the string
 */

function longestWord(sentence) {
  return sentence.split(' ').reduce((acc, curr) => {
    if (curr.length > acc.length) {
      return curr
    }
    return acc
  }, [])
}

console.log(longestWord('Walking talking having breakfast at tiffany\'s '))

File: easy/longest-word/longest-word.py
# Given a sentence, return the longest word in the string
def longest_word(sentence):
  sentence_list = sentence.split(' ')
  longest = ''
  for word in sentence_list:
    if len(word) > len(longest):
      longest = word
  
  return longest

print(longest_word('Walking talking having breakfast at tiffany\'s'))